Exception: "<pending_xref py:class=\"ModelError\" py:module=\"azure.iot.modelsrepository.exceptions\"\
  \ refdoc=\"azure.iot.modelsrepository\" refdomain=\"py\" refexplicit=\"False\" reftarget=\"\
  Exception\" reftype=\"class\" refwarn=\"False\">\n    <literal classes=\"xref py\
  \ py-class\">\n        Exception\n"
object: "<pending_xref py:class=\"ModelsRepositoryClient\" py:module=\"azure.iot.modelsrepository\"\
  \ refdoc=\"azure.iot.modelsrepository\" refdomain=\"py\" refexplicit=\"False\" reftarget=\"\
  object\" reftype=\"class\" refwarn=\"False\">\n    <literal classes=\"xref py py-class\"\
  >\n        object\n"
